### DISCUSSION TITLE
Windows Domain User accounts can install software using Winget. Can this be restricted to only Admin accounts

### DISCUSSION THREAD

**Discussion by loaderladdy-work** [microsoft/winget-cli#3383] [General]
I did do a brief search of this GH repo on the subject but couldn't find a similar question so am asking it here.

In our corp environment we have a need to manage how software is installed so we know we maintain our software inventories in compliance with software licensing Ts&Cs.

We found that it is possible for a user with only Windows Domain Admin rights to use winget to install software from both the Microsoft Store and winget package repository.

Is there documentation on how to approach locking down winget's config so only trusted software management processes, aka. Intune / MECM etc. can manage installation of winget packages?

Thanks in anticipation of a reply

Pete

**Comment by Trenly** [comment]
There are some [Group Policies](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/manage-windows-package-manager-with-group-policy/ba-p/2346322) available, but I'm not sure they meet your requirements

**Reply by loaderladdy-work** [reply]
thank you @Trenly I'll take a look and provide some feedback for you 👍
